src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+  position: sticky;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.logo {
+  position: absolute;
+  left: 50px;
+  top: 10px;
+}
+
+.nav {
+  height: 60px;
+  align-content: center;
+  margin-right: 50px;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  padding: 0;
+}
+
+.nav__item {
+  display: block;
+  margin: 0 10px;
+}
+
+.nav__link {
+  display: block;
+  align-self: center;
+  font-family: inherit;
+  font-style: normal;
```

Comment:
font-style: normal it's default value, so no need to set it

Reply:
Fixed